src/modules/commander/commander_params.c

```diff
@@ -256,18 +256,17 @@ PARAM_DEFINE_INT32(COM_RC_ARM_HYST, 1000);
  * automatically disarmed in case a landing situation has been detected during this period.
  *
  * The vehicle will also auto-disarm right after arming if it has not even flown, however the time
- * will be longer by a factor of 5.
+ * will always be 10 seconds such that the pilot has enough time to take off.
  *
- * A value of zero means that automatic disarming is disabled.
+ * A negative value means that automatic disarming triggered by landing detection is disabled.
  *
  * @group Commander
- * @min 0
+ * @min -1
  * @max 20
  * @unit s
- * @decimal 0
- * @increment 1
+ * @decimal 2
  */
-PARAM_DEFINE_INT32(COM_DISARM_LAND, 0);
+PARAM_DEFINE_FLOAT(COM_DISARM_LAND, -1.0f);
```

Comment:
FYI this will reset the parameter to default for anyone that's bothered to configured it. In this case I think we can live with it, but it's something to be aware of.

Reply:
This lines does not come alone. In this pr there is a feature I implemented which allows to set auto disarm to less than one second (making it float).
see https://github.com/PX4/Firmware/pull/8175/files#diff-1d51a813dcb26f46616f71f28aacc70dR2247
This requires the parameter to be -1 by default because of float comparison so the change is needed. But thanks for pointing out the user perspective side-effects.

Reply:
are these changes still valid?  they seem out of scope of the title of the PR.